tests/namespace_delete_test.go

```diff
@@ -82,16 +83,26 @@ func dynamicConfig() map[dynamicconfig.Key]interface{} {
 func (s *namespaceTestSuite) SetupSuite() {
 	s.logger = log.NewTestLogger()
 
-	clusterConfig, err := GetTestClusterConfig("testdata/integration_namespace_cluster.yaml")
-	s.Require().NoError(err)
+	var clusterConfig *TestClusterConfig
+	if TestFlags.PersistenceDriver == mysql.PluginNameV8 ||
+		TestFlags.PersistenceDriver == postgresql.PluginNameV12 ||
+		TestFlags.PersistenceDriver == sqlite.PluginName {
+		var err error
+		clusterConfig, err = GetTestClusterConfig("testdata/integration_test_cluster.yaml")
+		s.Require().NoError(err)
```

Comment:
Don't you need to setup the database? Or is it setup somewhere else?

Reply:
Yes, both DBs are already setup here. ES index also should be created earlier. This needs to be refactored.